index.js

```diff
@@ -2,27 +2,6 @@
 
 const globals = {};
 
-const getGlobal = property => {
```

Comment:
This could break some edge cases, including the scenario from https://github.com/sindresorhus/ky/pull/203 and https://github.com/sindresorhus/ky/issues/202. Basically, I've seen cases where some of the globals we rely on were actually defined on different objects (i.e. not true globals), particularly in mocked test environments.

That said, we're less reliant on these globals since the last time I had any issues and now that Node.js actually has `globalThis` and I can do `globalThis.window = globalThis`... maybe it won't be a problem? I'm not entirely sure. I'm 👍 on simplifying this, but let's definitely mark it as a breaking change.

Reply:
Let's try and see. I would prefer for us to not have to have lots of workarounds for weird use-cases.